CSS/components/grids.css

```diff
@@ -0,0 +1,17 @@
+.grid-wrap-2-2{
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-rows: 1fr 1fr ;
+    grid-template-columns:  1fr 1fr;
+}
+.grid-wrap-3-3{
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-rows: 1fr 1fr  1fr;
+    grid-template-columns:  1fr 1fr 1fr;
```

Comment:
Just s suggestion to use repeat(3,1fr) here

Reply:
Thank you for pointing out , refactored the code 👍